pkg/storage/buckets/store.go

```diff
@@ -40,8 +43,7 @@ type ListItem struct {
 
 // BucketStore contains objects store
```

Comment:
this comment is out of date now that objects.Store is removed

Reply:
Good catch, I will update all the comments once the code is sorted out